### ISSUE TITLE
Why the yaml file specified in env tag are searched in the relative path from the Snakefile?

### ISSUE THREAD

**Issue by kokitsuyuzaki** [snakemake/snakemake#544]
Why the yaml file specified in env tag are searched in the relative path from the Snakefile?

For example, if I write a Snakefile as follows,
input, output, benchmark, log, shell can be specified if the Snakefile were moved to the other directory, but only conda must be set in the same directory of Snakefile.

```bash
rule all
	input:
		'data/mydata.txt'
	output:
		'output/myoutput.txt'
	conda:
		'envs/myenv.yaml'
	benchmark:
		'benchmarks/myprogram.txt'
	log:
		'logs/myprogram.log'
	shell:
		'src/myprogram.sh'
```

**Comment by dangom** [comment]
I can confirm that this is confusing. Started learning snakemake a couple days ago and couldn't understand why the inconsistency.
Any hints how and where this could be addressed?

**Comment by dangom** [comment]
One workaround while the bug isn't fixed it to add a symbolic link in the rules directly pointing back, as in:
`ln -s .. workflow` run from the rules directory.
I tested and it works.
I think this bug stems from the new recommended best practices suggesting putting everything into a "workflow" directory.

**Comment by Hocnonsense** [comment]
> I think this bug stems from the new recommended best practices suggesting putting everything into a "workflow" directory.

Yes, "input, output, benchmark, log" are files action on by the workflow, but "snakefile, script, conda" are parts of the workflow.
